.github/ISSUE_TEMPLATE/bug.yml

```diff
@@ -0,0 +1,113 @@
+---
+description: Report a bug
+labels: [bug]
+name: Bug
+title: "<SCOPE>: <SUMMARY>"
+
+body:
+  - type: checkboxes
+
+    attributes:
+      label: I assert that this issue is relevant for Stylix
+
+      description: >-
+        If you need help or are unsure whether this is a Stylix bug, please
+        consider reading the [Documentation](https://stylix.danth.me) or asking
+        for help in a [GitHub
+        Discussion](https://github.com/danth/stylix/discussions) or the [Stylix
+        Matrix Room](https://matrix.to/#/#stylix:danth.me) before opening an
+        issue.
+
+      options:
+        - label: >-
+            I assert that this is a bug and not a user error or support
+            request.
+
+          required: true
+
+        - label: >-
+            I assert that this is not a duplicate of an [existing
+            issue](https://github.com/danth/stylix/issues).
+
+          required: true
+
+  - type: textarea
+
+    attributes:
+      description: >-
+        Describe the issue in detail. Consider including a minimal reproducible
+        example, logs, or screenshots.
+
+      label: Description
+
+      placeholder: Type your description here...
```

Comment:
The blank template uses a single character for the triple dots:

![image](https://github.com/user-attachments/assets/274ab22d-3497-41db-89a6-be1be8932259)

Here is the relevant HTML content:

```html
placeholder="Type your description here…"
```

We might want to do the same.

Apply this to all relevant templates.